src/app/rosetta/lib/block.ml

```diff
@@ -443,14 +416,11 @@ WITH RECURSIVE chain AS (
 
     let query =
       Caqti_request.collect Caqti_type.int typ
-        {| SELECT u.id, u.type, u.fee_payer_id, u.source_id, u.receiver_id, u.fee_token, u.token, u.nonce, u.amount, u.fee,
+        {| SELECT u.id, u.typ, u.fee_payer_id, u.source_id, u.receiver_id, u.nonce, u.amount, u.fee,
```

Comment:
should this be `command_type`?

Reply:
Updated in [cdd78a3](https://github.com/MinaProtocol/mina/pull/11960/commits/cdd78a3f2f00841fa23a2f32d563b618b4db5103). It would be nice if we could coordinate better, so that we don't step on each other's toes like that in the future. I'll try to raise this issue with management to see how it can be helped.